src/helpers/userHelper.js

```diff
@@ -171,6 +172,14 @@ const userHelper = function () {
           ? `<a href="${mediaUrl}">${mediaUrl}</a>`
           : 'Not provided!';
 
+        const googleDocLinkValue = adminLinks 
+        ? adminLinks.find(link => link.Name === 'Google Doc') 
```

Comment:
I have a suggestion for an edit:
**This line:** adminLinks.find(link => link.Name === 'Google Doc')
**Should be:** adminLinks.find(link => link.Name === 'Google Doc' &&  link.Link)
A Check for Link value is needed in case "Google Doc" name exist but the Link is empty.
Ex:
```
"adminLinks": [
    {
      "Name": "Google Doc",
      "Link": ""
    },]
```
This adjustment resolves the issue shown in the picture.
 
![Screenshot 2024-02-07 184250](https://github.com/OneCommunityGlobal/HGNRest/assets/49083865/775daf5d-b21e-4528-9209-a8cc9c82803b)

Reply:
Hello, @abdel-lall! Thank you for your review! I agree with your suggestions and have maded the necessary changes to the implementation, Requesting your approval upon testing the updated PR. Thank you!